exchange/howto/run-seednode.adoc

```diff
@@ -4,108 +4,306 @@
+=== Payment
 
+A seed node operator has the right to:
 
-=== Duties of the seed node operator
+- file a compensation request over 200 BSQ for setting up a seed node
+- file a monthly compensation request. We define 50 BSQ per month as an appropriate compensation per seed node per month.
+
+[NOTE]
+====
+With the advent of the DAO and the subsequent need of another 400GB of ROM, the numbers here are currently under discussion. Until new figures are decided on, state your expenses for running the server and add the 50 BSQ on top to get your sum for compensation.
+====
+
+== Getting in Touch
 
-The operator of a seed node needs to be responsive in case of seed node software updates as well to OS updates. If there are connectivity issues he needs to investigate and if required upgrade his server (RAM, CPU). We recommend 2-4 GB of RAM for one seed node to allow 30 connections. He should check the error logs and the node log file (in the data directory) occasionally. He has to have a professional level of operational security to operate the node.
-Operators need to subscribe to the Bisq Slack channel 'seednode' and act if their node reports errors.
+Bisq uses Slack to communicate via chat. As a seed node operator, you are required to
 
+- subscribe to the Bisq Slack channel `bisq-seednode` and `bisq-monitor`
+- the `bisq-seednode` channel is the place where updates, issues, countermeasures, heads-ups, ... are discussed. If you encounter a problem with your seed node and cannot solve it by yourself, this is the place to report to (with a specific question, logs, ...). A developer will get back to you.
+- the `bisq-monitor` channel is the place where issues with seed nodes are reported, either manually or by our monitoring service. If your seed node is mentioned for having an issue, you are required to react.
+- please be responsive when addressed
 
 == System requirements for hosting machine
 
-. Min. 2 GB of RAM
-. 10 GB disk space (SSD)
-. 2 TB network traffic
-. UPS (uninterruptible power supply)
-. Uptime of > 99.9%
+- Min. 4 GB of RAM
+- 400 GB of disk space
+- 2 TB network traffic
+- Uptime of > 99.9%
+- > 10000 file handles available (check `ulimit -Sn`)
+
+And please try to stay away from cheap hosting providers! They tend to not care about QOS, the hardware is slow, the occasionally even turned a service off without warning - ask us how we know...
+
+## Get a seed node up and running
+
+You can run your seed node in many ways. The guide you are currently reading applies to a (debian-based) unix system with systemd available. Furthermore, we assume that only one seed node is operated per host. Please be aware that the guide is only a guide and not a step by step copy and paste how to, as we ignore for example the need for the occational root access and file permissions, swap, general opsec, among other things. In case your host differs from our reference system, we are sure you can handle yourself.
```

Comment:
```suggestion
You can run your seed node in many ways. The guide you are currently reading applies to a (debian-based) unix system with systemd available. Furthermore, we assume that only one seed node is operated per host. Please be aware that the guide is only a guide and not a step by step copy and paste how to, as we ignore for example the need for the occasional root access and file permissions, swap, general opsec, among other things. In case your host differs from our reference system, we are sure you can handle yourself.
```

Reply:
What exactly is your suggestion here?

Reply:
For some reason it is not picking up my diff. I was suggesting to change "occational" -> "occasional".